shared-ui/styled-components/day-background/background.style.ts

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+
+const StyledDayBackground = styled.img`
+  width: 100%;
+  position: absolute;
+  z-index: -1;
+  top: 0;
```

Comment:
Instead setting top + bottom to 0, can we create a global.css file in /shared-ui/style and add 
`body {
  margin: 0;
}
`
This directly gets rid of the body margin 
<img width="473" alt="Screen Shot 2022-09-11 at 2 12 01 PM" src="https://user-images.githubusercontent.com/59738880/189542670-0adec92d-f603-4ed9-90af-a251c1197ed5.png">

You'll need to import the css file into the index.tsx of main-site for it to apply